suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraph.tsx

```diff
@@ -55,7 +49,7 @@ export const PortfolioGraph = forwardRef<PortfolioGraphRef>((_props, ref) => {
 
     return (
         <VStack spacing="large" testID="@home/portfolio/graph">
-            {isAnyMainnetAccountPresent ? <PortfolioGraphHeader /> : null}
+            <PortfolioGraphHeader />
```

Comment:
Is removal of `isAnyMainnetAccountPresent` intentional?

Reply:
definitely not, probably bad rebase

Reply:
👌 check the extra `{' '}`